common/src/lib.rs

```diff
@@ -580,7 +583,7 @@ pub enum StringEncoding {
 
 // This is NonZeroU128 because we regularly store Option<NodeId>.
```

Comment:
Suggest replacing this non-doc comment with the following docstring:

```rust
/// The inner type of a [`NodeId`]
///
/// This is currently 128 bits to accommodate UUIDs, but may change or be configurable,
/// to reduce memory usage. It is NonZero because it is commonly used in an `Option`.
```